src/Console/Helper/TaskRunnerHelper.php

```diff
@@ -160,6 +163,14 @@ private function returnWarningMessages($output, array $warningMessages)
         }
     }
 
+    /**
+     * @param OutputInterface $output
+     */
+    private function returnAdditionalInfo(OutputInterface $output)
+    {
+        is_null($this->config->getAdditionalInfo()) ?: $output->writeln($this->config->getAdditionalInfo());
```

Comment:
This syntax is a bit confusing. Could you just use an if instead?

Reply:
This should resolve this conversation https://github.com/phpro/grumphp/pull/539/commits/d31d927dafc46cf194ac5f12cf81af4a39605b9a